File: crates/deslop/tests/fixtures/fused-golden-python/shape_gamma.py
class TelemetryDescriptor:
    stream = "gamma-telemetry"
    window = 47
    sink = "https://gamma.example.com/telemetry"

    def build(self, device):
        return self.sink + "/" + device + "/" + self.stream + "?window=" + str(self.window)

File: crates/deslop/tests/fixtures/fused-golden-php/shape_gamma.php
<?php

class TelemetryDescriptor
{
    public string $stream = 'gamma-telemetry';
    public int $window = 47;
    public string $sink = 'https://gamma.example.com/telemetry';

    public function build(string $device): string
    {
        return $this->sink . '/' . $device . '/' . $this->stream . '?window=gamma';
    }
}
